thrift/events.thrift

```diff
@@ -1,3 +1,5 @@
+include "contentatom.thrift"
+
 typedef string ContentAtomID
 
 enum EventType { PUBLISH, UPDATE, TAKEDOWN }
```

Comment:
Is there definitely a need to differentiate between publishing and updating? It might make a lot of code simpler if we don't have to.